tcpclient/connection_test.go

```diff
@@ -0,0 +1,41 @@
+package tcpclient
+
+import (
+	"testing"
+	"time"
+)
+
+func TestGetConnectionStatus(t *testing.T) {
```

Comment:
https://stackoverflow.com/a/6197432

Reply:
be or not to be, this is the question 😄 
what do you think? get rid of it?